docs/beaver-builder/developer/tutorials-guides/remove-rename-tools-menu-items.md

```diff
@@ -0,0 +1,51 @@
+---
+id: remove-rename-tools-menu-items
+title: Rename or remove Tools menu items
+sidebar_label: Rename or remove Tools menu items
+---
+
+This tutorial gives you details of how to customize the [Tools menu](/beaver-builder/getting-started/bb-editor-basics/tools-menu.md), such as removing or renaming menu items.
+
+
+## Remove menu items
+
+The code example below shows how to remove menu items and includes some of the main Tools menu items. You can find all Tools menu `$views` item ID's by searching for `// Tools` in the */bb-plugin/classes/class-fl-builder.php* file. 
```

Comment:
UI controls and labels are in bold, so it would be **Tools** menu in the body (not bolded in titles or headings though). 
Filenames and pathnames are in italics.